### ISSUE TITLE
Rendered doc for enum generated by macros

### ISSUE THREAD

**Issue by MarcoCicognani** [intellij-rust/intellij-rust#6594]
## Environment

* Plugin v0.3.139.3575-203-nightly
* rustc 1.51.0-nightly (257becbfe 2020-12-27)
* CLion 2020.3
* Manjaro Linux 

## Problem description
Hi,
I currently work to a Rust project and, to reduce boilerplate code, i use a macro to generate `enum` with `From`, `Into` & `Display` support.

The problem comes when i use the rendered view of CLion to show the documentation comments, that i write over the `enum` and his variants.

CLion shows `Not available` message instead of the rendered text of the comment.

## Steps to reproduce
I use the following macro that i write by myself
`
#[macro_export]
macro_rules! c_handy_enum {
	{$(#[$Comments:meta])*
	pub enum $EnumName:ident : $ToFromType:ident {
	    $( $(#[$Meta:meta])* $Variant:ident = $Index:expr => $String:tt,)*
	}} => {
	    $(#[$Comments])*
	    #[repr($ToFromType)]
		#[derive(Clone, Copy, PartialEq, Eq, Debug)]
		pub enum $EnumName {
			$( $(#[$Meta])* $Variant = $Index,)*
		}
	}

        // other code for the trait implementations
}
`
when i use it

`
c_handy_enum! {
    /// Hi To every one
    pub enum AnEnum: u8 {
        /// Doc comment to variant
        Variant0 = 0 => "Variant0",
    }
}
`
CLion displays this enabling rendered documentation comments:
![Selection_067](https://user-images.githubusercontent.com/20948691/103386008-1f76e900-4afd-11eb-98a5-001626cf47f6.png)

**Comment by fee1-dead** [comment]
When looking at `syn` documentations, it says not available as well. However, when hovering over a reference to the enum/struct, it will display the documentation.

**Comment by MarcoCicognani** [comment]
@fee1-dead yeah I noticed, but I would like that the rendered documentation is displayed directly over the `enum`/`struct`
